InstallSpinnaker.sh

```diff
@@ -689,7 +689,7 @@ if [[ "$INSTALL_CASSANDRA" != "false" ]]; then
 fi
 
 ## Packer
-mkdir /tmp/packer && pushd /tmp/packer
+mkdir -p /tmp/packer && pushd /tmp/packer
```

Comment:
Perhaps this should be something like:

PACKER_DIR=$(mktemp -d)
pushd $PACKER_DIR
..
rm -rf $PACKER_DIR

And I see there are other mkdir calls in this script, which are susceptable to the same issue.
Maybe wrap the whole script in a TEMPDIR=$(mktemp -d installspinnaker.XXXX) up top and rm -rf $TEMPDIR below and have each of the internal mkdir calls (including this) use $TEMPDIR instead of /tmp